tests/phpunit/api/v3/EmailTest.php

```diff
@@ -376,4 +376,38 @@ public function testReplaceEmailWithId() {
     $this->assertEquals('1-2@example.com', $get['values'][$emailID]['email']);
   }
 
+  public function testEmailOnHold() {
+    $params = array();
+    $params_change = array();
+    $params = array(
+      'contact_id' => $this->_contactID,
+      'email' => 'api@a-team.com',
+      'on_hold' => '2',
+    );
+    $result = $this->callAPIAndDocument('email', 'create', $params, __FUNCTION__, __FILE__);
+    $this->assertEquals(1, $result['count']);
```

Comment:
You should do a $this->callAPISuccess('Email', 'get', ['id' => $result['id]); to make sure you have the stored data